src/extension/python/python.c

```diff
@@ -18,7 +18,7 @@ static FilteredSysnum filtered_sysnums[] = {
 };
 
 /* build by swig */
-extern void init_proot(void);
+extern void PyInit__proot(void);
```

Comment:
Why did this change? Just curious :)

Reply:
This seems like something Swig changed as they have 
```
#if PY_VERSION_HEX >= 0x03000000
#  define SWIG_init    PyInit__proot

#else
#  define SWIG_init    init_proot

#endif
```
in the generated files. 

For better backward compatibility with folks having Python 2.x, I'll probably add the same logic in here as well